docs/modules/ROOT/pages/tutorials/demo-app/appcat-nextcloud.adoc

```diff
@@ -0,0 +1,129 @@
+= Deploy https://products.vshn.ch/appcat/nextcloud.html[VSHNNextcloud] to {product}
+
+image:logos/nextcloud.svg[role="related thumb right",alt="Service logo nextcloud",width=120,height=120] This tutorial explains how to run https://nextcloud.com/[Nextcloud] with link:++https://products.vshn.ch/appcat/nextcloud.html[VSHNNextcloud]++[VSHNNextcloud] on {product} using the pipeline from https://www.vshn.ch/en/solutions/appflow/[Appflow].
+
+== Requirements
+
+* Access to {product}
+* A https://github.com[GitHub] account
+
+== Add the https://products.vshn.ch/appcat/index.html[VSHN Application Catalog] claim
+
+To provision Nextcloud, we need a https://products.vshn.ch/appcat/nextcloud.html[VSHNNextcloud] claim. The documentation for VSHNNextcloud can be found here: https://docs.appcat.ch/vshn-managed/nextcloud/index.html[Nextcloud by VSHN].
+We start with a very simple setup:
+
+.VSHNNextcloud
+[source,yaml]
+----
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNNextcloud
+metadata:
+  name: appcat-demo-nextcloud-appuio-simple # <1>
+  namespace: vshn2-demo-nextcloud-appuio-simple-prod # <2>
+spec:
+  parameters:
+    service:
+      fqdn:
+      - appuio-nextcloud-demo.apps.cloudscale-lpg-2.appuio.cloud # <3>
+      version: "32" # <4>
+    size: # <5>
+      plan: standard-2
+  writeConnectionSecretToRef:
+    name: nextcloud-creds # <6>
+----
+<1> Instance name
+<2> The namespace where the object will be created
+<3> Your fully qualified domain name. You should be able to change the DNS record for this domain.
+<4> Nextcloud version
+<5> Size of the Nextcloud instance. See https://docs.appcat.ch/vshn-managed/nextcloud/plans.html[Plans and Sizing] for more information.
+<6> Secret where the connection details are provisioned. No secret with this name should exist in this namespace before creation.
+
+== Put the claim under version control
+
+To keep track of what's deployed, we put this claim into our Git repository.
+For that, you need to create a https://github.com/new[new Git repository] on GitHub.
+Now, we store the claim from before under `helm/templates/nextcloud.yaml`.
```

Comment:
Does https://docs.appcat.ch/references/argocd.html also apply if the claim is wrapped in a helm chart? Never tested that.

Reply:
We let helm try to delete the resources in the deployments we did until now.
The helm uninstall fails in this case because of the deletion protection.
If you want to uninstall a helm chart with appcat claims you must disable the deletion protection before.